build_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num_commits=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+current_commit=$(git rev-parse --short=7 HEAD)
+version="0.1.$num_commits-$current_commit"
+opm_version="1.14.0"
+
+# Login to docker
+docker_conf="$PWD/.docker"
+mkdir -p "$docker_conf"
+docker_cmd="docker --config=$docker_conf"
+
+$docker_cmd login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+
+# Find the CSV version from the previous bundle
+$docker_cmd pull $BUNDLE_IMAGE_NAME:latest && exists=1 || exists=0
+
+if [ $exists -eq 1 ]; then
+  docker create --name="tmp_$$" $BUNDLE_IMAGE_NAME:latest sh
+  tmp_dir=$(mktemp -d -t sa-XXXXXXXXXX)
+  pushd $tmp_dir
+    docker export tmp_$$ | tar -xf -
+    prev_version=`find . -name *.clusterserviceversion.* | xargs cat - | yq r - metadata.name`
+  popd
+  rm -rf $tmp_dir
+  docker rm tmp_$$
+fi
+
+# Build/push the new bundle
+pushd deploy/bundle
+  IMAGE=$BUNDLE_IMAGE_NAME IMAGE_TAG=$current_commit VERSION=$version REPLACE_VERSION=$prev_version make bundle
```

Comment:
Does this work ok if `prev_version' is empty? If so, I would make it more explicit, as it can be confusing

Reply:
It does.  That was the sed logic we were trying to reshape to work across linux and mac